Samples/WindowsCopilotRuntime/cs-wpf-dotnet/WCRforWPF/MainWindow.xaml.cs

```diff
@@ -86,92 +88,157 @@ private async void ProcessFile_Click(object sender, RoutedEventArgs e)
         this.FileContent.Text = "Extracting text...";
         this.Description.Text = "(waiting)";
 
-        // First, extract text from the image using the OCR model
-        var ocr = await EnsureTextRecognizerAsync();
-        if (ocr == null)
+        try
         {
-            this.FileContent.Text = "No OCR model available";
-            return;
+            this.FileContent.Text = "Loading AI models...";
+            this.Description.Text = "Loading AI models...";
```

Comment:
Given my aptitude for typos... Can the code be 'simplified' to a single string?   
            this.FileContent.Text = "Loading AI models...";
            this.Description.Text = this.FileContent.Text;